lib/elixir/src/elixir_errors.erl

```diff
@@ -252,19 +252,20 @@ prefix(error) ->
   end.
 
 env_format(Meta, #{file := EnvFile} = E) ->
-  {File, Line} = meta_location(Meta, EnvFile),
+  {File, Position} = meta_location(Meta, EnvFile),
+  Line = ?line(Position),
```

Comment:
This line is now wrong, because it is not respecting the `meta_keep` call from `meta_location`. We should make `meta_location` return `{File, Line, Position}`. :)

Reply:
Is it? In my understanding in `meta_location`, the line from `meta_keep` is kept if it finds anything, and it goes into the resulting `Position`, maybe I'm missing something here.

Also, looking at `meta_location` I think I forgot to pass the column from meta along with the `meta_keep` line, if any, right?

Reply:
Oh, sorry, you're right, we are reading it from the position. However, it feels weird to read something we just put in a list so returning {File, Line, Position} is better anyway?

Regarding the keep, you are right, it never has columns (and I am not sure if it should, separate discussion indeed).

Reply:
I'm ok with that, I just feel it a bit weird because now the `compile_error` cases that call `meta_location` will have to ignore this line because they don't need it, so maybe it's best to explicitly fetch the line on the case where it's needed

Reply:
I see your point. I am glad to leave it as is then. Thank you :)